tests/cases/conformance/classes/propertyMemberDeclarations/accessorsOverrideProperty9.ts

```diff
@@ -0,0 +1,49 @@
+// @strict: true
+// @target: es2017
+// #41347, based on microsoft/rushstack
+
+// Mixin utilities
+export type Constructor<T = {}> = new (...args: any[]) => T;
+export type PropertiesOf<T> = { [K in keyof T]: T[K] };
+
+interface IApiItemConstructor extends Constructor<ApiItem>, PropertiesOf<typeof ApiItem> {}
+
+// Base class
+class ApiItem {
+  public get members(): ReadonlyArray<ApiItem> {
+    return [];
+  }
+}
+
+// Normal subclass
+class ApiEnumMember extends ApiItem {
+}
+
+// Mixin base class
+interface ApiItemContainerMixin extends ApiItem {
+  readonly members: ReadonlyArray<ApiItem>;
```

Comment:
I'm curious why this is `readonly members: ReadonlyArray<ApiItem>;` and not `get members(): ReadonlyArray<ApiItem>;`. Is it purely to support pre-TS 4.0, which wouldn't be able to parse the declaration file? 

4.0 has been available for over a year and a half, and 41347 was filed only two months after 4.0 shipped. If we are weakening restrictions to improve backcompat support, I'd want to be sure the backcompat support is warranted. If api-extractor still supports consumers using < TS 4.0, then I could see this being valuable.

Reply:
@octogonz, is pre-TS 4.0 support still critical to api-extractor package consumers?

Reply:
It's still an error because of the way union/intersection properties are created. If I bring back the commit with create-unions-of-accessors-as-accessors, your suggestion might work. Let me check.

Reply:
Nope, the changed check is still required. (It's not that much looser either; it just allows any declaration of a synthetic symbol instead of only valueDeclaration).